README.md

```diff
@@ -1,3 +1,20 @@
-# Map Out Hate
+## Map Out Hate
 
-The purpose of this project is to create a data set for instances of micro-aggresions and discrimination. 
+The purpose of this project is to create a data set for micro-aggressions and discrimination. It is a safe space for people to report incidents that happen to them. It also provides resources for users to take further action and prepare themselves for various situations. When a user reports pins are posted on a map and they are able to visualize all other reports that have come in. Click [here](http://map-out-hate.herokuapp.com/) to explore the application.
+
+# Tools Used for Creation
+This application utilizes Facebook's third party authentication to log in. Google Map's API for the map that allows users to visualize incidents. AnjularJS was used on the front end with sass and Bootstrap for styling. The backend uses NodeJS. Testing was done with Karma, Mocha, and Chai. The build process runts through Grunt. Heroku automatically deploys the site through the master branch on GItHub.
```

Comment:
Didn't mention your database, and there's a small type on "GItHub" (capitalization).